### DISCUSSION TITLE
能否给 `--no-delay` 选项加个开关？

### DISCUSSION THREAD

**Discussion by pexcn** [shadowsocks/shadowsocks-android#2710] [Ideas]
新的版本里改了名，现在叫 `--tcp-no-delay`
https://github.com/shadowsocks/shadowsocks-rust/blob/49b7004d7565de98a24b49657443b19c03365a8f/bin/sslocal.rs#L68

**Comment by Kel2021** [comment]
[]()![Uploading Screenshot_20210424-031554.png - # [ ] …](

**Reply by Kel2021** [reply]
- [ ] # - [ ] Thank you

**Reply by pexcn** [reply]
where the screenshot?
